3drepo/3drepo.io
# Issue 4731 - Status should be shown at all times

This fixes #4731 

#### Description
Templates get always fetched before tickets so that, when fetching the tickets, all the ones that are missing the status use the default one from the template

#### Test cases
Access old tickets that are not supposed to have a status and check that the default one from template is applied.
In the viewer:
- open the tickets list and expand one of those tickets
- open the viewer directly to that ticket expanded (using the search param)

In tabular view:
- access the table via ticket selection;
- access the table via refresh from the table view;
- expand a ticket from the table view
